scene/resources/resource_format_text.cpp

```diff
@@ -2047,6 +2086,48 @@ Error ResourceFormatSaverTextInstance::save(const String &p_path, const Ref<Reso
 			}
 			f->store_line("[editable path=\"" + editable_instances[i].operator String().c_escape() + "\"]");
 		}
+
+		// Store Overrides.
+		for (int i = 0; i < state->get_override_count(); i++) {
+			if (i == 0) {
+				f->store_line("");
+			}
+			String header = "[override";
+			NodePath path = state->get_override_path(i);
+			Vector<StringName> groups = state->get_override_groups(i);
+			if (path != NodePath()) {
+				header += " path=\"" + String(path.simplified()).c_escape() + "\"";
+			}
+			if (groups.size()) {
```

Comment:
```suggestion
			if (!groups.is_empty()) {
```
You could also remove the comment, because it's the same one as elsewhere.